src/cli.c

```diff
@@ -224,6 +226,14 @@ int main(int argc, char *argv[])
                     defines[numDefines - 1].name = SDL_malloc(len);
                     SDL_utf8strlcpy(defines[numDefines - 1].name, (const char *)argv[i] + 2, len);
                 }
+            } else if (SDL_strcmp(arg, "--msl-version") == 0) {
+                if (i + 1 >= argc) {
+                    SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "%s requires an argument", arg);
+                    print_help();
+                    return 1;
+                }
+                i += 1;
+                mslVersion = SDL_atoi(argv[i]);
```

Comment:
It's kind of awkward to take in the version int directly because clients will have to calculate SPVC_MAKE_MSL_VERSION themselves. Maybe the property should just be a "major.minor.patch" string and then the internal method can parse and convert it.

Reply:
Done! Supplying e.g. `--msl-version 2.1.0` is the way this works now.